pkg/client/clientset/versioned/typed/crd/v1alpha1/fake/fake_crd_client.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Antrea Authors
```

Comment:
ditto

Reply:
These files are generated automatically by `make codegen` so I think the change is expected.